donate/payments/utils.py

```diff
@@ -12,3 +13,16 @@ def freeze_transaction_details_for_session(details):
 @lru_cache(maxsize=1000)
 def get_currency_info(currency):
     return CURRENCIES[currency].copy()
+
+
+def get_suggested_monthly_upgrade(currency, single_amount):
+    info = get_currency_info(currency)
+
+    # Check if we have upgrades manually specified for this currency, and if so use that
+    for tier in info.get('monthlyUpgrade', []):
+        if Decimal(single_amount) >= tier['min']:
+            return Decimal(tier['value'])
+
+    # If we don't have manual values, or they didn't return a suggestion, then
```

Comment:
@alanmoo let me know if this sounds OK to you. Use `monthlyUpgrade` where set, otherwise fall back to 10% of the single amount.

Reply:
Yes, but we need a minimum amount, which is partially why we don’t ask until we hit a threshold. Otherwise there’s potential for recurring small donations to cost us money.  (We also need that for the custom amount field. If that wasn’t built in, I’m happy to file it)

Reply:
I'm merging this to unblock Nick on templates, and moving this discussion to the feature ticket so that we can make the appropriate adjusments before closing that.